src/components/productDetails/list.jsx

```diff
@@ -27,7 +28,7 @@ const List = ({
   var [outfitsReviews, setOutfitsReviews] = useState([]);
 
   // Set Local State For Conditionally Rendered Products
-  let initialShownIndices = listName === "Outfit" ? [0, 1, 2] : [0, 1, 2, 3];
+  let initialShownIndices = listName === "Outfit" ? [0, 1, 2, 3] : [0, 1, 2, 3];
   var [shownIndices, setShownIndices] = useState(initialShownIndices);
 
```

Comment:
i am assuming the addition of the ',3' insertion here is what you were referencing for the faux next item tile. very cool